### PR TITLE
Feature/multiple messages

### PR DESCRIPTION
**Pull request by grigi**
FB states that one should iterate over `entry` and `messaging` due to automatic batching.

### CODE REVIEW

**Comment by grigi** [comment]
I'm concerned that any one of X messages being unsupported or invalid will raise an `UnsupportedMessage` exception. What do you propose we do there?

I see the options as:
1) Leave as is, and lose messages
2) Silence the failures (and lose error messages)
3) Log errors and then resume (and attach Raven to the logger to get Sentry messages)
4) Return a tuple of messages and errors, let `handle_raw_inbound_message` handle the exception. (And hand the errors to Vumi? Does it even do that?)

File: vxmessenger/transport.py [hunk 1]
```diff
@@ -183,45 +191,50 @@ def handle_raw_inbound_message(self, message_id, request):
...
-                    'mid': page.mid,
-                }
-            },
-            helper_metadata={
-                'messenger': helper_metadata
-            })
+        self.log.info("MessengerTransport inbound %r" % (pages,))
+        if errors:
+            for error in errors:
+                self.log.error(error)
```
Comment (smn) [inline vxmessenger/transport.py]: Major nitpick but the `if errors` isn't necessary since errors will always be a list? Feel free to ignore.

**Comment by smn** [comment]
:+1: thanks for doing this!

File: vxmessenger/transport.py (hunk 1 above)
Comment (grigi) [inline vxmessenger/transport.py]: wow, that is very silly of me. It should be either:
if errors:
    log errors
or:
for error in errors:
    log error

Which would you prefer? I think I started with the second one, then changed badly to the first.

**Pull request merged**